### PR TITLE
feat(editor): Add descriptive header to projects /workflow

### PR DESCRIPTION
**Pull request by r00gm**
## Summary

Add header above tabs in project /workflow view
image
image
image
image
image

## Related Linear tickets, Github issues, and Community forum posts

[PAY-1997](https://linear.app/n8n/issue/PAY-1997/update-project-tab-view)

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/components/layouts/ResourcesListLayout.vue [hunk 1]
```diff
@@ -339,10 +343,36 @@ export default defineComponent({
 			}
 		});
 
+		const headerIcon = computed(() => {
+			if (projectsStore.currentProject?.type === 'personal') {
```
Comment (cstuncsik) [inline packages/editor-ui/src/components/layouts/ResourcesListLayout.vue:347]: Please use`ProjectTypes` everywhere from `packages/editor-ui/src/types/projects.types.ts` when comapring project types.
E.g.: `projectsStore.currentProject?.type === ProjectTypes.Personal`

File: packages/editor-ui/src/views/WorkflowsView.test.ts
Comment (r00gm) [inline packages/editor-ui/src/views/WorkflowsView.test.ts:1]: @cstuncsik one example of how one test break when it's component code has not being changed

File: packages/editor-ui/src/views/ProjectSettings.vue [hunk 2]
```diff
@@ -247,6 +248,31 @@ watch(
 	{ immediate: true },
 );
 
+const headerIcon = computed(() => {
+	if (projectsStore.currentProject?.type === 'personal') {
```
Comment (cstuncsik) [inline packages/editor-ui/src/views/ProjectSettings.vue:252]:
```suggestion
	if (projectsStore.currentProject?.type === ProjectTypes.Personal) {
```

File: packages/editor-ui/src/views/ProjectSettings.vue [hunk 3]
```diff
@@ -247,6 +248,31 @@ watch(
...
+		return 'layer-group';
+	} else {
+		return 'home';
+	}
+});
+
+const projectName = computed(() => {
+	if (!projectsStore.currentProject) {
+		return locale.baseText('projects.menu.home');
+	} else if (projectsStore.currentProject.type === 'personal') {
```
Comment (cstuncsik) [inline packages/editor-ui/src/views/ProjectSettings.vue:264]:
```suggestion
	} else if (projectsStore.currentProject.type === ProjectTypes.Personal) {
```

**Review by cstuncsik — DISMISSED** [review]

**Review by cstuncsik — DISMISSED** [review]

**Review by cstuncsik — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.66.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.66.0)

**Pull request merged**